tests/cmd/main_test.py

```diff
@@ -7,17 +7,18 @@
 
 import mock
 
+import undebt.examples.method_to_function
 from undebt.cmd.main import _exit_fail_upon_error
 from undebt.cmd.main import _load_text
 from undebt.cmd.main import _process_file
 from undebt.cmd.main import _write_result_text
 from undebt.cmd.main import main
-from undebt.examples import method_to_function
 
 
-tests_inputs_directory = os.path.join(os.path.dirname(os.path.dirname(__file__)), "inputs")
+method_to_function = undebt.examples.method_to_function
```

Comment:
```
from undebt.examples import method_to_function
```
instead of
```
import undebt.examples.method_to_function
method_to_function = undebt.examples.method_to_function
```
is what paiwei is going for here, I think, and if so I agree.